app/ui/src/Pages/Project/pages/Overview/Overview.tsx

```diff
@@ -32,6 +32,10 @@ function Overview({ openedProject }: Props) {
           <div className={styles.sectionTitle}>PROJECT NAME</div>
           <div className={styles.name}>{openedProject.name}</div>
         </div>
+        <div className={styles.section}>
+          <div className={styles.sectionTitle}>PROJECT ID</div>
+          <div className={styles.name}>{openedProject.id}</div>
```

Comment:
I think Id should be slightly smaller than name
```suggestion
          <div className={styles.id}>{openedProject.id}</div>
```